docs/yubikey_setup.rst

```diff
@@ -21,7 +21,7 @@ Download and Launch the YubiKey Personalization Tool
 ----------------------------------------------------
 
 #. Start Tails. At the log in-screen, choose the option to allow an
-   administrator passphrase.
```

Comment:
s/administrator/administration/ this is what Tails display, we can't change that. Not easily at least.